.travis.yml

```diff
@@ -0,0 +1,36 @@
+sudo: false
+language: python
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=py27
+    - python: 3.4
+      env: TOXENV=py34
+    - python: 3.5
+      env: TOXENV=py35
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.6
+      env: TOXENV=bandit
+    - python: 3.6
+      env: TOXENV=doc8
+    - python: 3.6
+      env: TOXENV=readme
+# pending reorg of deserialize_header
```

Comment:
As discussed, flag this as TODO and link to some tracker so we don't forget?

Reply:
Issue opened to fix the underlying issue: https://github.com/awslabs/aws-encryption-sdk-python/issues/17